src/containers/ResetPassword/ResetPassword.js

```diff
@@ -0,0 +1,146 @@
+import Container from "@material-ui/core/Container";
+import { makeStyles } from "@material-ui/core/styles";
+
+function ResetPassword() {
+  const classes = useStyles();
+
+  // it should return new password to backend
+  // it will be updated later on!
+  const onSubmit = () => {
+    return null;
+  };
+
+  return (
+    <Container component="main" maxWidth="sm">
+      <CssBaseline />
+      <div className={classes.resetPassword}>
+        <Avatar className={classes.avatar}>
+          <LockOutlinedIcon />
+        </Avatar>
+        <Typography component="h1" variant="h5">
+          Reset Password
+        </Typography>
+        <Form
+          onSubmit={onSubmit}
+          validate={(values) => {
+            const errors = {};
+            if (!values.password) {
+              errors.password = "Required";
+            }
+            if (!values.confirm) {
+              errors.confirm = "Required";
+            } else if (values.confirm !== values.password) {
+              errors.confirm = "Must match";
+            }
+            return errors;
+          }}
+          render={({ handleSubmit, form, values }) => (
+            <form onSubmit={handleSubmit} className={classes.form}>
+              <Field name="password">
+                {({ input, meta }) => (
+                  <div>
+                    <label className={classes.label}>New Password</label>
+                    <input
+                      {...input}
+                      type="password"
+                      placeholder="new password"
+                      className={classes.password}
+                    />
+                    {meta.error && meta.touched && (
+                      <span className={classes.error}>{meta.error}</span>
+                    )}
+                  </div>
+                )}
+              </Field>
+              <Field name="confirm">
+                {({ input, meta }) => (
+                  <div>
+                    <label className={classes.label}>
+                      Confirm New Password
+                    </label>
+                    <input
+                      {...input}
+                      type="password"
+                      placeholder="confirm new password"
+                      className={classes.confirmPassword}
+                    />
+                    {meta.error && meta.touched && (
+                      <span className={classes.error}>{meta.error}</span>
+                    )}
+                  </div>
+                )}
+              </Field>
+              <Button
+                type="submit"
+                fullWidth
+                variant="contained"
+                color="primary"
+                className={classes.submit}
+              >
+                Submit
+              </Button>
+            </form>
+          )}
+        />
+      </div>
+      <Box mt={8}>
+        <Copyright />
+      </Box>
+    </Container>
+  );
+}
+
+function Copyright() {
```

Comment:
Why not separate this component into a separate file?

Reply:
Yes i have the component in the forget password and register branch you can get it use it.